pkg/api/metrics.go

```diff
@@ -138,6 +143,10 @@ func (hs *HTTPServer) QueryMetrics(c *models.ReqContext, reqDto dtos.MetricReque
 
 	resp, err := tsdb.HandleRequest(c.Req.Context(), ds, request)
 	if err != nil {
+		if errors.Is(err, tsdberrs.BadRequest{}) {
```

Comment:
I cannot see any code referring to creating a `tsdberrs.BadRequest` so this should probably be removed.